src/java/utils/JsonHelper.java

```diff
@@ -0,0 +1,17 @@
+package utils;
+
+import net.sf.json.JSONObject;
+
+/**
+ *
```

Comment:
A short description of the scope of the members of this class will be helpful in setting limits on this or determining when it may combine with other similar utility classes.